server/routes/index.js

```diff
@@ -0,0 +1,27 @@
+import client from '../helpers/redis';
+
+import daysBetween from '../helpers/daysBetween';
+
+export default (app) => {
+  app.get('/worker-status', (req, res) => {
+
```

Comment:
Adding some tests for this API endpoint is going to be awesome!

Reply:
Just for the sake of the folks that are following this thread, a chore has been created to adress this issue https://www.pivotaltracker.com/story/show/162152989